cmd/spec/path.yaml

```diff
@@ -133,3 +133,54 @@ paths:
             $ref: "#/components/responses/NotFound"
           '500':
             $ref: "#/components/responses/InternalError"
+  /reservations:
+    get:
+      operationId: getReservationsList
+      description: >
+        A reservation is a way to activate a job, keeps all data needed for a job to start.
+        This operation returns list of all reservations for particular account.
+      responses:
+        '200':
+          description: 'Returned on success.'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/v1.Reservation'
+        "500":
+          $ref: '#/components/responses/InternalError'
+  /reservations/aws:
+    post:
+      operationId: createAwsReservation
+      description: >
+        A reservation is a way to activate a job, keeps all data needed for a job to start.
+        An AWS reservation is a reservation created for an AWS job. 
+      responses:
+        '200':
+          description: 'Returned on success.'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/v1.AWSReservation'
+        "500":
+          $ref: '#/components/responses/InternalError'
+  /reservations/noop:
+    post:
+      operationId: createNoopReservation
+      description: >
+        A reservation is a way to activate a job, keeps all data needed for a job to start.
+        A Noop reservation actually does nothing and immediately finish background job. 
+      responses:
+        '200':
+          description: 'Returned on success.'
+          content:
+            application/json:
+              schema:
+                type: array
```

Comment:
is that supposed to be array? @lzap @ezr-ondrej

Reply:
I guess we should respond with empty body, but does not matter much, best would be to check what we actually return in the request and just go with that.

Reply:
this is what we are returning an AWSReservation

Reply:
Oh I totally forgot, this is the problem that @avitova had to solve because `sql.NullXYZ` does not nicely map to JSON `null`. Schema generator does not support that, so it actually generates invalid schema (empty maps). The solution was to split model and payload and we need to do the same here, unless someone else figures out better way.